loadtest/user/userentity/actions.go

```diff
@@ -156,6 +156,14 @@ func (ue *UserEntity) GetPostsForChannel(channelId string, page, perPage int) er
 	return ue.store.SetPosts(posts)
 }
 
+func (ue *UserEntity) GetPostsBefore(channelId, postId string, page, perPage int) error {
+	_, resp := ue.client.GetPostsBefore(channelId, postId, page, perPage, "")
+	if resp.Error != nil {
+		return resp.Error
+	}
+	return nil
```

Comment:
Don't add it to the storage until we have an append method in the storage because its a partial result

Reply:
Thinking it twice, isn't the `MemStore.SetPosts` already what we need here? We are simply merging posts (adding or updating).

Reply:
Oh right, we're adding by postId index instead of replacing the whole post slice. Adding that part then :)